lib/Page.js

```diff
@@ -44,9 +44,9 @@ class Page extends EventEmitter {
     if (ignoreHTTPSErrors)
       await client.send('Security.setOverrideCertificateErrors', {override: true});
     const page = new Page(client, ignoreHTTPSErrors, screenshotTaskQueue);
-    await page.goto('about:blank');
     // Initialize default page size.
     await page.setViewport({width: 800, height: 600});
+    await page._frameManager.fetchResourceTree();
```

Comment:
You are fetching resource tree after initializing Runtime - that way you won't get execution contexts mapped to the frames.